tests/scripts/docker/run_tests.sh

```diff
@@ -47,7 +47,7 @@ fi
 
 # CASS_DRIVER_NO_EXTENSIONS is set so we don't build the Cassandra C-extensions,
 # as this can take several minutes
-docker build --build-arg PYTHON_IMAGE=${1/-/:} -t apm-agent-python:${1} . # replace - with : to get the correct docker image
+
```

Comment:
Given a PR that changes the `Dockerfile`, how will it be possible to change the PR in the CI? I'm not sure whether that's something we do often.

Reply:
I just added this a couple of minutes ago.

https://github.com/elastic/apm-agent-python/pull/1792/commits/1cd662e614829d9062c7e560b47eaa02b5864df4

this way we can utilise the cache and still build it on a individual commit.

Reply:
https://github.com/elastic/apm-agent-python/pull/1792#issuecomment-1496992110